core/server/services/newsletters/service.js

```diff
@@ -161,14 +161,14 @@ class NewslettersService {
             // subscribe members that have an existing subscription to an active newsletter
             const memberIds = await this.MemberModel.fetchAllSubscribed(_.pick(options, 'transacting'));
 
-            newsletter.meta = newsletter.meta || {};
-            newsletter.meta.opted_in_member_count = memberIds.length;
-
             if (memberIds.length) {
                 debug(`Found ${memberIds.length} members to subscribe`);
 
                 await newsletter.subscribeMembersById(memberIds, options);
+                newsletter = await this.NewsletterModel.findOne({id: newsletter.id}, {...options, require: true});
```

Comment:
We now reload the newsletter model twice ([here](https://github.com/TryGhost/Ghost/pull/14858/files#diff-d61df6455ea81ae89293ed9306ecc5f763bd281ae0e4cc4c588bd3552ab35400R155)) - can we avoid that?